edk2toollib/uefi/authenticated_variables_structure_support.py

```diff
@@ -53,32 +69,33 @@ def __init__(self,
             (Exception): Invalid Parameters
         """
         if (decodefs is not None):
-            self.PopulateFromFileStream(decodefs, decodesize)
+            self.decode(decodefs, decodesize)
         elif (createfs is not None):
             # create a new one
-            self.SignatureOwner = sigowner
-            start = createfs.tell()  # should be 0 but maybe this filestream has other things at the head
+            self.signature_owner = sigowner
```

Comment:
I think with these changes you'll need to do python's getter/setter changes so you can do deprecation warnings on the variables that you are changing the name of. Hopefully no one is accessing this data directly but you just never know.

https://www.geeksforgeeks.org/getter-and-setter-in-python/